clean_code/algorithms/pattern_algorithms.py

```diff
@@ -0,0 +1,65 @@
+from bisect import insort
+
+
+"""This module supplies fast algorithms for pattern-related searches:
+   longest palindrome, biggest elements in a list."""
+
+def find_n_biggest_numbers(numbers, amount=20):
+    """return the nth biggest elements in the list"""
+    if len(numbers) > amount:
+        biggest_numbers = sorted(numbers[:amount])
+    else:
+        return numbers
```

Comment:
I think it will be more clear if this `if` will handle only one thing...
```python
if len(numbers) <= amount
    return numbers

biggest_numbers = sorted(numbers[:amount])
```